server/methods/email.js

```diff
@@ -74,7 +74,7 @@ Meteor.methods({
 
     delete settings.password;
 
-    Logger.info(settings, "Email settings updated");
+    Logger.debug(settings, "Email settings updated");
```

Comment:
This happens quite infrequently.  It may be worth having stuff like this at info level.